linter/whitelists/de.txt

```diff
@@ -328,6 +328,7 @@ Responsive
 responsiven
 Rewrite
 Rewriting
+Rocksolid
```

Comment:
Actually, it should be enough if "Rocksolid" is on the default whitelist, isn't it?

Reply:
yes, it should. But Travis throwed an error and that's why I added it to de.txt as well.

Reply:
Ah, okay.